agents/src/ipc/proc_job_executor.ts

```diff
@@ -94,9 +94,9 @@ export class ProcJobExecutor extends JobExecutor {
       }
     };
     this.#proc!.on('message', listener);
-    this.#proc!.on('error', (err) => {
+    this.#proc!.on('error', () => {
       if (this.#closing) return;
-      this.#logger.child({ err }).warn('job process exited unexpectedly');
+      this.#logger.warn('job process exited unexpectedly');
```

Comment:
I think it would be good to at least keep the error message for debugging purposes though in cases where it might not be that exact error that's causing it? 

We can include "look for the actual error further up in the logs" or something like that